lib/cocoapods-core/podfile/target_definition.rb

```diff
@@ -610,6 +631,17 @@ def set_platform!(name, target = nil)
         set_platform(name, target)
       end
 
+      # Stores the Swift version requirements to be used for this target.
+      #
+      # @param   [String, Version, Array<String>, Array<Version>] requirements
```

Comment:
passing in an array here won't work unless you `flatten` it

Reply:
fixed